src/assets/highlightsIcon.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="17px" height="17px" viewBox="0 0 17 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>D3F63949-593E-441A-BC31-85CECF26BF85</title>
```

Comment:
I think this is either not required or should be something readable

Reply:
This came with it when I downloaded them from the designs, but yeah whatever, removed